rubygem-sprockets/rubygem-sprockets.spec

```diff
@@ -0,0 +1,163 @@
+Source1: sprockets-%{version}-tests.tgz
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ror}rubygem(rack) > 1
+Requires: %{?scl_prefix_ror}rubygem(rack) < 3
+Requires: %{?scl_prefix}rubygem(concurrent-ruby) >= 1
+Requires: %{?scl_prefix}rubygem(concurrent-ruby) < 2
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.3
+BuildRequires: %{?scl_prefix_ruby}rubygem(rake)
+BuildRequires: %{?scl_prefix_ruby}rubygem(minitest)
+BuildRequires: %{?scl_prefix_ror}rubygem(therubyracer)
+BuildRequires: %{?scl_prefix_ror}rubygem(uglifier)
+BuildRequires: %{?scl_prefix_ror}rubygem(coffee-script)
+BuildRequires: %{?scl_prefix_ror}rubygem(execjs)
+BuildRequires: %{?scl_prefix_ror}rubygem(rack-test)
+BuildRequires: %{?scl_prefix}rubygem(sass)
+BuildRequires: %{?scl_prefix}rubygem(ejs)
+BuildRequires: %{?scl_prefix}rubygem(nokogiri)
+BuildRequires: %{?scl_prefix}rubygem(concurrent-ruby)
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%description
+Sprockets is a Rack-based asset packaging system that concatenates and serves
+JavaScript, CoffeeScript, CSS, LESS, Sass, and SCSS.
+
+%package doc
+Summary:    Documentation for %{pkg_name}
+Group:      Documentation
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch:  noarch
+
+%description doc
+Documentation for %{pkg_name}
+
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+
+mkdir -p %{buildroot}%{_bindir}
+cp -a .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+# Run the test suite
+
+%check
+pushd .%{gem_instdir}
+tar xzf %{SOURCE1}
+
+# We don't have rubygem(closure-compiler) yet.
+# https://bugzilla.redhat.com/show_bug.cgi?id=725733
+mv test/test_closure_compressor.rb{,.disabled}
+mv lib/sprockets/autoload/closure.rb{,.disabled}
+sed -i '/:Closure/ s/^/#/' lib/sprockets/autoload.rb
+
+# We don't have rubygem(eco) yet.
+mv test/test_eco_processor.rb{,.disabled}
+mv lib/sprockets/autoload/eco.rb{,.disabled}
+sed -i '/:Eco/ s/^/#/' lib/sprockets/autoload.rb
+sed -i '/test "eco templates" do/,/^  end/ s/^/#/' test/test_environment.rb
+
+# We don't have rubygem(yui-compressor) yet.
+# https://bugzilla.redhat.com/show_bug.cgi?id=725768
+mv test/test_yui_compressor.rb{,.disabled}
+mv lib/sprockets/autoload/yui.rb{,.disabled}
+sed -i '/:YUI/ s/^/#/' lib/sprockets/autoload.rb
+
+# Required by TestPathUtils#test_find_upwards test.
+touch Gemfile
+
+%{?scl:scl enable %{scl} %{scl_v8} - << \EOF}
+RUBYOPT=-Ilib:test ruby -e 'Dir.glob "./test/**/test_*.rb", &method(:require)' \
+  | grep "763 runs, 3278 assertions, 0 failures, 3 errors, 1 skips"
```

Comment:
I've been trying to get minitest to run in `--verbose` mode to print the skipped tests here but I couldn't because I can't pass the argument. I tried to run `rake test TESTOPTS='--verbose'` to no avail. 

In any case, the skip locally on f23 is the one I mentioned before about Windows, so I assume it's the same.